sdk/Pulumi/Deployment/Deployment_RegisterResource.cs

```diff
@@ -5,6 +5,7 @@
 using System.Linq;
 using System.Threading.Tasks;
 using Google.Protobuf.WellKnownTypes;
+using Microsoft.Extensions.Options;
```

Comment:
What's this for?

Reply:
Not needed, must have auto-added at some point in the earlier code.